packages/vuetify/src/components/VChip/VChip.tsx

```diff
@@ -146,7 +146,7 @@ export const VChip = defineComponent({
           v-ripple={ [isClickable && props.ripple, null] }
           onClick={ isClickable && onClickFunc }
         >
-          { genOverlays(isClickable, 'v-chip') }
+          { genOverlays(true, 'v-chip') }
```

Comment:
#10115

Should probably use the same logic as VListItem

Reply:
Thank you for feedback.
I thought it would be highlighted even if it was unclickable.
I fixed to highlight only when clickable.